### DISCUSSION TITLE
Welcome to Train_Transformers_with_INT4 Discussions!

### DISCUSSION THREAD

**Discussion by haochengxi** [haochengxi/Train_Transformers_with_INT4#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shoveller86** [comment]
Hi, the experiments in paper includes ""ViT" benchmark, but I could not find it in source code?

**Comment by xjz-ai** [comment]
请问使用2080ti显卡复现这个实验时，修改using SmArch = cutlass::arch::Sm80; 及其他参数 为sm75时导致编译不通过该怎么解决呢，或者使用4090显卡可以吗，4090也不是安培架构。 刚好没有3090显卡
